apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-editor.tsx

```diff
@@ -0,0 +1,50 @@
+import { RiPencilRuler2Line } from 'react-icons/ri';
+import { UiSchemaGroupEnum, type UiSchema } from '@novu/shared';
+
+import { getComponentByType } from '@/components/workflow-editor/steps/component-utils';
+
+const avatarKey = 'avatar';
+const subjectKey = 'subject';
+const bodyKey = 'body';
+const redirectKey = 'redirect';
+const primaryActionKey = 'primaryAction';
+const secondaryActionKey = 'secondaryAction';
+
+export const InAppEditor = ({ uiSchema }: { uiSchema?: UiSchema }) => {
```

Comment:
The component that renders the In-App Editor UI form dynamically based on the `uiSchema`.